README.md

```diff
@@ -1,182 +1,163 @@
 # JavaScript Data Types
-
-In this lesson, we'll cover some of the most commonly used data types in JavaScript.
+In this lesson, we'll cover all of the basic data types in JavaScript.
 
 ## Objectives
-1. Define "data type"
-2. Explain JavaScript's data types
-3. Use the `typeof` operator
+1. Define 'data type'.
+2. Introduce the `typeof` operator.
+3. Provide a high-level overview of JavaScript's seven basic data types.
+4. Discuss interactions between data of various types in JavaScript.
 
 ## What is a data type?
+At the machine level, all data on a computer are bits — 1s and 0s. Humans, it turns out, prefer not to work so close to the metal, so we developed _data types_ for representing different pieces of information. JavaScript contains a number of operators (`+`, `!`, `<=`, etc.) and reserved words (`function`, `for`, `debugger`, etc.), which are reserved by the language for a specific purpose. _Everything_ else is data. And every piece of data falls into one of JavaScript's seven data types: numbers, strings, booleans, symbols, objects, `null`, and `undefined`.
 
-At the machine level, all data on a computer is bits — 1s and 0s. Humans, it turns out, prefer not to work so close to the metal, so we have _data types_ for describing different bits of information. Data types give us a quick way of understanding how we can operate on a given bit of data.
-
-## Math!
-
-Imagine you're building a new-fangled version of those toys where you have to match the shape with the hole it fits into.
+***NOTE***: Throughout this lesson, we'll use the `typeof` operator to give us an idea of what data types we're dealing with. `typeof` accepts one [operand](https://en.wikipedia.org/wiki/Operand), the piece of data that we'd like to know the _type of_.
```

Comment:
Don't introduce operand here. Just call it an argument. I don't want folks going down a Wikipedia rabbit hole!

Also introducing new terms is tough for students. They aren't sure if it's important to really understand it or not